File: cpp/bubbleSort.cpp
#include <iostream>
using namespace std;

void bubbleSort(int * array, int size);
	
int main(){
	
	return 0;
}

void bubbleSort(int * array, int size){
    for(int i = 0; i < size - 1; i++){
        for(int j = 0; j < size - i - 1; j++){
            if(array[j+1] < array[j]){
                int tmp = array[j + 1];
                array[j + 1] = array[j];
                array[j] = tmp;
            }   
        }   
    }   
} 

File: python/bubble_sort.py

def bubble_sort(ls):
    for i in range(0, len(ls) - 1):
        for j in range(0, len(ls) - 1 - i):
            if ls[j] > ls[j + 1]:
                tmp = ls[j]
                ls[j] = ls[j + 1]
                ls[j + 1] = tmp

if __name__ == '__main__':
    arr = [2, 4, 5, 7, 8, 10]
    bubble_sort(arr)
    print(arr)
